traitsui/handler.py

```diff
@@ -543,11 +543,10 @@ class Controller ( Handler ):
 
     #-- HasTraits Method Overrides ---------------------------------------------
 
-    def __init__ ( self, model = None, **metadata ):
+    def __init__ ( self, **traits):
```

Comment:
With this modification, the `__init__` isn't needed at all.  However the original version avoids an initial trait change firing on the setting of the model, which may be intentional.